lib/rules/all.js

```diff
@@ -24,8 +62,9 @@ for (const rule in rules) {
         ImportDeclaration(node) {
           if (node.source.value === 'lodash') {
             // ex: import { indexOf } from 'lodash';
+            // ex: import { indexOf as x } from 'lodash';
             node.specifiers.forEach(specifier => {
-              if (specifier.type === 'ImportSpecifier' && specifier.local.name === rule) {
+              if (specifier.type === 'ImportSpecifier' && specifier.imported.name === rule) {
```

Comment:
This change fixes the `import { indexOf as x } from 'lodash';` case.
Only `import { indexOf } from 'lodash';` would be notified.